src/World.cpp

```diff
@@ -324,6 +324,33 @@ void cWorld::SetNextBlockTick(int a_BlockX, int a_BlockY, int a_BlockZ)
 
 
 
+bool cWorld::SetSpawn(double a_X, double a_Y, double a_Z)
+{
+		cIniFile IniFile;
+
+		IniFile.ReadFile(m_IniFileName);
+		IniFile.SetValueF("SpawnPosition", "X", a_X);
+		IniFile.SetValueF("SpawnPosition", "Y", a_Y);
+		IniFile.SetValueF("SpawnPosition", "Z", a_Z);
+		if (IniFile.WriteFile(m_IniFileName))
+		{
+			m_SpawnX = a_X;
```

Comment:
I don't have an opinion in either way. Once you get a file write error, having a correct spawn coord is the least of your problems; perhaps a log message would be good in this case, though.